.github/workflows/stale.yaml

```diff
@@ -0,0 +1,42 @@
+name: 'Close stale issues and PRs'
+
+on:
+  push:
+    paths:
+      - .github/workflows/stale.yml
+    branches-ignore:
+      - dependabot/**
+  schedule:
+    # Once every day at midnight UTC. Source: https://crontab.guru/every-day
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # Setting this variable to 'true' will cause the
+          # action to run in a dry-run mode
+          debug-only: ${{ vars.DEBUG_ONLY == 'true' }}
+          stale-issue-message: >
+            This issue has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs.
+          close-issue-message: >
+            Closing this issue after a prolonged period of inactivity. If this
+            issue is still present in the latest release, please feel free to
+            create a new issue with up-to-date information.
+          stale-pr-message: >
+            This pull request has been automatically marked as stale because it has not had
+            recent activity. It will be closed if no further activity occurs.
```

Comment:
Should we also mention the total number of days in which the PR closes for more clarity?

Reply:
Done